docs/api/sql/Function.md

```diff
@@ -106,10 +106,10 @@ Introduction:
 Transform the Spatial Reference System / Coordinate Reference System of A, from SourceCRS to TargetCRS
 
 !!!note
-	By default, this function uses lat/lon order. You can use ==ST_FlipCoordinates== to swap X and Y.
+        By default, this function uses lat/lon order. You can use ==ST_FlipCoordinates== to swap X and Y.
```

Comment:
You still changed the doc. This should be a tab. No change.

Reply:
Just did the right commit. Should be good now